addons/mail/static/src/components/composer_text_input/composer_text_input.js

```diff
@@ -162,8 +164,10 @@ class ComposerTextInput extends Component {
      * @private
      */
     _updateHeight() {
-        this._textareaRef.el.style.height = "0px";
-        this._textareaRef.el.style.height = (this._textareaRef.el.scrollHeight) + "px";
+        // console.log(this._textareaRef.el);
+        this._textareaDummyRef.el.style.height = "0px";
+        this._textareaRef.el.style.height = (this._textareaDummyRef.el.scrollHeight) + "px";
+        this._textareaDummyRef.el.style.height = (this._textareaDummyRef.el.scrollHeight) + "px";
```

Comment:
I think you might be able to permanently leave the dummy at 0px height, as long as content is updated and height says at 0, the fake scrollHeight will read the correct value no?

Reply:
you are right, it work's the same.